filebeat/channel/outlet.go

```diff
@@ -88,3 +89,7 @@ func (o *Outlet) OnEventSignal(event *input.Event) bool {
 		return true
 	}
 }
+
+func (o *Outlet) Copy() prospector.Outlet {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported method Outlet.Copy should have comment or be unexported